c2_buffers/Android.bp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017-2023 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+cc_library_static {
+    name: "libmfx_c2_buffers",
+
+    static_libs: [
+    ],
+
+    shared_libs: [
+        "libexpat",
+        "libgralloctypes",
+        "libstagefright_foundation",
+        "android.hardware.media.bufferpool@2.0",
+        "android.hardware.graphics.bufferqueue@2.0",
+        "android.hardware.graphics.common@1.2",
+    ],
+
+    defaults: ["mfx_c2_defaults"],
+
+    srcs: [
+        "src/*.cpp",
+    ],
+
+    cflags: [
+    ],
+
+    local_include_dirs: [
+        "include",
+    ],
+
+    include_dirs: [
+        "vendor/intel/mediasdk_c2/c2_components/include",
+        "vendor/intel/mediasdk_c2/c2_utils/include",
+        "frameworks/av/media/codec2/vndk/include",
+        "hardware/intel/external/libva",
```

Comment:
Can we have libva_headers included in mfx_c2_defaults? As I see libva_headers is declared in va android.bp. Then we can remove libva folder in include_dirs here.

Thanks,
Tianmi

Reply:
yes, I just added that.